src/Hl7.Fhir.ElementModel/TypedElementOnSourceNode.cs

```diff
@@ -3,7 +3,7 @@
  * See the file CONTRIBUTORS for details.
  * 
  * This file is licensed under the BSD 3-Clause license
- * available at https://github.com/FirelyTeam/fhir-net-api/blob/master/LICENSE
+ * available at https://github.com/ewoutkramer/fhir-net-api/blob/master/LICENSE
```

Comment:
Oops: can you change this back to FirelyTeam?

Reply:
That must be coming from a merge, yes.